bigquery_etl/cli/query.py

```diff
@@ -117,7 +118,41 @@ def query(ctx):
     default=False,
     is_flag=True,
 )
-def create(name, sql_dir, project_id, owner, init):
+@click.option(
+    "--dag",
+    "-dag",
+    help=(
+        "Name of the DAG the query should be scheduled under. "
+        "To see available DAGs run `bqetl dag info`. "
+        "To create a new DAG run `bqetl dag create`. To skip the "
+        " automated scheduling use option --no_schedule."
+    ),
+)
+@click.option(
+    "--use_default_dag",
+    "-default-dag",
+    help=(
+        "Using this option automatically schedules the query in DAG"
+        " `bqetl_default`. This DAG is classified tier_3, and the lowest"
+        " level of service from Data Engineering."
+    ),
+    default=False,
+    is_flag=True,
+)
```

Comment:
It would be easier for the developers, but they might not be aware of it (it would be visible in the PR review). During infraday we preferred this option for awareness. However I am gathering more opinions and we can go for the option that people prefer (see below in Sean's comment)